### PR TITLE
2nd Eclipse Tractus-X Community Days Contribution

### PR DESCRIPTION
**Pull request by ClosedSourcerer**
## Description
- Increased Deployability
- Fixed minor Issues
- TRG 8 will be taken care off with a seperate PR into the Upstream Repository.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by carslen** [comment]
> The build is failing due to prettier. Need to rebase once #72 merged in the main branch again

Please do a new PR with these changes as this PR has been reverted after it has been merged accidentaly.

**Review by carslen — CHANGES_REQUESTED** [review]
Please fix [failing ECA check](https://api.eclipse.org/git/eca/status/gh/eclipse-tractusx/demand-capacity-mgmt/79).

Also update `DEPENDENCIES_BACKEND` file as it seems that new deps has been added.

**Comment by nitin-vavdiya** [comment]
> > The build is failing due to prettier. Need to rebase once #72 merged in the main branch again
> 
> Please do a new PR with these changes as this PR has been reverted after it has been merged accidentaly.

New PR: https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/80

File: demand-capacity-mgmt-backend/src/main/resources/db/migration/V16__initial_data.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (evegufy) [inline demand-capacity-mgmt-backend/src/main/resources/db/migration/V16__initial_data.sql:1]: Please add file header

**Review by evegufy — CHANGES_REQUESTED** [review]
Hi @ClosedSourcerer @nitin-vavdiya nice contribution in general!
Agree with all comments from @carslen and one small comment from my side

File: demand-capacity-mgmt-backend/src/main/resources/db/migration/V16__initial_data.sql (hunk 1 above)
Comment (nitin-vavdiya) [reply demand-capacity-mgmt-backend/src/main/resources/db/migration/V16__initial_data.sql]: Fixed with: https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/79/commits/889cbaa5306f96acf87b9fde5e6c81e131dd8415

**Comment by nitin-vavdiya** [comment]
> Please fix [failing ECA check](https://api.eclipse.org/git/eca/status/gh/eclipse-tractusx/demand-capacity-mgmt/79).
> 
> Also update `DEPENDENCIES_BACKEND` file as it seems that new deps has been added.

Thanks for reviewing this PR.
Regarding `DEPENDENCIES_BACKEND`, it will be added with [this PR](https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/80/files#diff-f3185f05c8d717818f75bbdd6a0db01f563bd1910d338bad684421fd1d0ee36c)

**Comment by carslen** [comment]
> > Please fix [failing ECA check](https://api.eclipse.org/git/eca/status/gh/eclipse-tractusx/demand-capacity-mgmt/79).
> > Also update `DEPENDENCIES_BACKEND` file as it seems that new deps has been added.
> 
> Thanks for reviewing this PR. Regarding `DEPENDENCIES_BACKEND`, it will be added with [this PR](https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/80/files#diff-f3185f05c8d717818f75bbdd6a0db01f563bd1910d338bad684421fd1d0ee36c)

Sorry to disagree, but if you introduce a new dependency with a PR, you need to update the DEPENCENCIES file accordingly. A reference to another PR is not valid.

**Review by evegufy — CHANGES_REQUESTED** [review]
There are suddenly 888 commits showing, could you please clean this up?
That's probably also the reason why GitGuardian comes up with those findings and once that's clean up, I assume the finding will be gone.
But in general, if GitGuardian reveals findings they need to documented as what they are in the incident management section (either real - and in that case they need to revoked immediately  - or false positive/test) of GitGuardian.

Also, please re-request the review once there should be reviewed again, otherwise reviewers can't be aware that the pull request should be reviewed again.

**Comment by nitin-vavdiya** [comment]
> There are suddenly 888 commits showing, could you please clean this up? That's probably also the reason why GitGuardian comes up with those findings and once that's clean up, I assume the finding will be gone. But in general, if GitGuardian reveals findings they need to documented as what they are in the incident management section (either real - and in that case they need to revoked immediately - or false positive/test) of GitGuardian.
> 
> Also, please re-request the review once there should be reviewed again, otherwise reviewers can't be aware that the pull request should be reviewed again.

I think I mashed up with the PR  and sorry for that
"There are suddenly 888 commits showing"  > I rebased this PR that's why I think 
Does closing this PR and creating fresh PR from my owned form repo make sense?

**Comment by evegufy** [comment]
> > There are suddenly 888 commits showing, could you please clean this up? That's probably also the reason why GitGuardian comes up with those findings and once that's clean up, I assume the finding will be gone. But in general, if GitGuardian reveals findings they need to documented as what they are in the incident management section (either real - and in that case they need to revoked immediately - or false positive/test) of GitGuardian.
> > Also, please re-request the review once there should be reviewed again, otherwise reviewers can't be aware that the pull request should be reviewed again.
> 
> I think I mashed up with the PR and sorry for that "There are suddenly 888 commits showing" > I rebased this PR that's why I think Does closing this PR and creating fresh PR from my owned form repo make sense?

Hi @nitin-vavdiya rebasing with 800 + commits to drop can be tedious, opening a new pull request is fine (please reference to this one in the description so that we don't loose the discussions).

**Comment by nitin-vavdiya** [comment]
> > > There are suddenly 888 commits showing, could you please clean this up? That's probably also the reason why GitGuardian comes up with those findings and once that's clean up, I assume the finding will be gone. But in general, if GitGuardian reveals findings they need to documented as what they are in the incident management section (either real - and in that case they need to revoked immediately - or false positive/test) of GitGuardian.
> > > Also, please re-request the review once there should be reviewed again, otherwise reviewers can't be aware that the pull request should be reviewed again.
> > 
> > 
> > I think I mashed up with the PR and sorry for that "There are suddenly 888 commits showing" > I rebased this PR that's why I think Does closing this PR and creating fresh PR from my owned form repo make sense?
> 
> Hi @nitin-vavdiya rebasing with 800 + commits to drop can be tedious, opening a new pull request is fine (please reference to this one in the description so that we don't loose the discussions).

New PR : https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/102
@Phil91 We can close this PR

**Pull request closed without merging**